src/asana/helpers.py

```diff
@@ -26,6 +26,8 @@
     "AttachmentData", "file_name file_url image_type"
 )
 
+StatusReason = collections.namedtuple("StatusReason", "isClosed reason")
```

Comment:
Is it Pythonic to us `isClosed` rather than `is_closed` here?

Reply:
absolutely not! Haven't touched Python in many weeks, sorry!

Reply:
@arthurborem-asana  -- I still think we should use `is_closed`

Reply:
or `is_complete` even